tests/fixtures/deprecated_symbols/codemod/register_pytree_node.py.out

```diff
@@ -0,0 +1,9 @@
+from torch.utils._pytree import register_pytree_node, _register_pytree_node
```

Comment:
I hoped it would no longer import _register_pytree_node...
(Not for this PR, but an issue for the future).